ampd/src/handlers/stacks_verify_msg.rs

```diff
@@ -0,0 +1,409 @@
+use std::collections::HashSet;
+use std::convert::TryInto;
+
+use async_trait::async_trait;
+use axelar_wasm_std::voting::{PollId, Vote};
+use cosmrs::cosmwasm::MsgExecuteContract;
+use cosmrs::tx::Msg;
+use cosmrs::Any;
+use error_stack::ResultExt;
+use events::Error::EventTypeMismatch;
+use events::Event;
+use events_derive::try_from;
+use futures::future;
+use router_api::ChainName;
+use serde::Deserialize;
+use tokio::sync::watch::Receiver;
+use tracing::info;
+use voting_verifier::msg::ExecuteMsg;
+
+use crate::event_processor::EventHandler;
+use crate::handlers::errors::Error;
+use crate::stacks::http_client::{Client, Transaction};
+use crate::stacks::verifier::verify_message;
+use crate::types::{Hash, TMAddress};
+
+type Result<T> = error_stack::Result<T, Error>;
+
+#[derive(Deserialize, Debug)]
+pub struct Message {
+    pub tx_id: Hash,
+    pub event_index: u32,
```

Comment:
We updated voting verifier recently to emit the message_id string in the event instead and deprecated the old fields. Now you can directly decode into the typed MsgIdFormat, see example [here](https://github.com/axelarnetwork/axelar-amplifier/blob/main/ampd/src/handlers/stellar_verify_msg.rs#L33). We'll be announcing this in the upcoming release's changelog. But you can update multiversX/stacks handlers to use this already. The new voting verifier contract will need to be deployed first so the new field is emitted for ampd to parse

Reply:
Done